core/cmd/app.go

```diff
@@ -300,6 +300,12 @@ func NewApp(s *Shell) *cli.App {
 			Usage:       "Commands for managing forwarder addresses.",
 			Subcommands: initFowardersSubCmds(s),
 		},
+		{
+			Name:        "chaintablecleanup",
```

Comment:
I think this might belong grouped under `chainlink node db <something>` - e.g. `delete-chain`?